test/unit_tests/test_ci_prepare.py

```diff
@@ -0,0 +1,27 @@
+import os
+from pathlib import Path
+from unittest import mock
+
+import pytest
+from exasol_integration_test_docker_environment.cli.options.system_options import DEFAULT_OUTPUT_DIRECTORY
+from exasol_integration_test_docker_environment.lib.base import luigi_log_config
+
+from exasol_script_languages_container_ci.lib.ci_prepare import CIPrepare
+
+
+@pytest.fixture(autouse=True)
+def mock_settings_env_vars():
+    with mock.patch.dict(os.environ, {}):
+        yield
```

Comment:
is wiping the entire `env` always necessary? (autouse)

Reply:
for these tests yes, the autouse is restricted to this module, if I understand pytest correct.